File: tournaments/isCorrectSentence/isCorrectSentence.cpp
bool isCorrectSentence(std::string inputString) {

  char leadChar = inputString[0],
       endChar = inputString[inputString.size() - 1];

  if ('A' <= leadChar && leadChar <= 'Z'
      && endChar == '.') {
    return true;
  }
  else {
    return false;
  }
}

File: tournaments/isCorrectSentence/isCorrectSentence.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub isCorrectSentence {
    my ($inputString) = @_;

    my @store = split '', $inputString;

    return 'A' le $store[0] && 'Z' ge $store[0] && '.' eq $store[@store - 1];
}
